Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Outlook.Grid/Classes/Formating/Params/BarParams.cs

```diff
@@ -27,15 +27,15 @@ public class BarParams : IFormatParams
         /// <summary>
         /// The bar color
         /// </summary>
-        public Color BarColour;
+        public Color _barColour;
```

Comment:
These are public.. Do not make them look like private fields !!

Reply:
Reverted